### ISSUE TITLE
New routes should be announced

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#10596]
If you switch from one screen to another, it should announce what the new screen is.

**Comment by Hixie** [comment]
In general for edge-triggered accessibility effects we're going to need a new API specifically for that purpose. The current accessibility stuff is entirely level-triggered (describe the current state, hand the state off to the OEM layer).

**Comment by goderbauer** [comment]
Closing in favor of https://github.com/flutter/flutter/issues/13704.
